src/core/lib/iomgr/tcp_client_cfstream.cc

```diff
@@ -140,10 +140,12 @@ static void OnOpen(void* arg, grpc_error_handle error) {
 
 static void ParseResolvedAddress(const grpc_resolved_address* addr,
                                  CFStringRef* host, int* port) {
-  std::string host_port = grpc_sockaddr_to_string(addr, true);
+  auto host_port = grpc_sockaddr_to_string_or_errormsg(addr, true);
```

Comment:
I don't see any such function.  Is this change left over from a previous iteration of this PR?

Reply:
Right, wasn't aware this is in a ifdef and was never compiled, fixed.